hackclub.com.yaml

```diff
@@ -735,6 +735,10 @@ cken:
   - ttl: 1
     type: A
     value: 52.191.165.18
+codeabit:
+  - ttl: 1
```

Comment:
TTL should be at least 600, 3600 is usually good

Reply:
done